golang/src/tvm_runtime_pack.cc

```diff
@@ -32,6 +32,7 @@
 #include "src/runtime/threading_backend.cc"
 #include "src/runtime/thread_pool.cc"
 #include "src/runtime/ndarray.cc"
+#include "src/runtime/memory_manager.cc"
```

Comment:
for now, do not include memory_manager.cc, consider add back once we have all the vm changes

Reply:
It doesn't link correctly without this line

Reply:
if you break the dependency of the NDArrary from memory_manager(move allocator aware alloca to allocator->Empty), you should be able to break the link dp